src/MiniCover/Commands/BaseCommand.cs

```diff
@@ -6,40 +6,41 @@
 
 namespace MiniCover.Commands
 {
-    internal abstract class BaseCommandLineApplication : CommandLineApplication
+    internal abstract class BaseCommand
```

Comment:
Since that command line library was created with fluent configuration in mind, I think it's safer to use the fluent configuration methods instead of inheritance.